### ISSUE TITLE
How to add execution pin like in UE4 blueprint rather than just value input/output pin?

### ISSUE THREAD

**Issue by nagasawamaki** [jagenjo/litegraph.js#501]
Would like to see how to add execution pin like in UE4 blueprint.

Is it possible?

Thanks

**Comment by atlasan** [comment]
Yes, that was been integrated, just drag an event connection (square) into a node, it will create an input onTrigger action slot and an output onExecuted event slot and change the mode of the node to On Trigger.
